lib/sheets_db/worksheet.rb

```diff
@@ -3,6 +3,8 @@
 
 module SheetsDB
   class Worksheet
+    class WorksheetColumnNotFoundError < StandardError; end
```

Comment:
This seems like a redundant error class name, and doesn't fit the style already established (of not including the parent class name in the error).  I'd prefer the full error class to be `SheetsDB::Worksheet::ColumnNotFoundError`; is there a reason you opted for this instead?